ssl/ssl_locl.h

```diff
@@ -570,9 +569,7 @@ struct ssl_session_st {
         size_t ecpointformats_len;
         unsigned char *ecpointformats; /* peer's list */
```

Comment:
Would it make sense to move also the ecpointformats?

Reply:
AFAICT there is no race condition with ecpointformats so it doen't *need* to move. However, there isn't a good reason to store this data in the session object, and therefore for consistency reasons it probably does make sense to move this one too.